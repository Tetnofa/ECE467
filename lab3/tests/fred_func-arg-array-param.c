void f1(int arr[]);

void f1(int arr[]) {
  
}
