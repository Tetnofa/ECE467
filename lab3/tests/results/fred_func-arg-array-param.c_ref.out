Program [useIO=0](1,0)
	Function[isProto=1]	Void(1,0)
		Identifier[name:f1](1,5)
		Parameter		Int*(1,8)
			Identifier[name:arr](1,12)
	Function[isProto=0]	Void(3,0)
		Identifier[name:f1](3,5)
		Parameter		Int*(3,8)
			Identifier[name:arr](3,12)
		Scope(3,19)
